var/spack/repos/builtin/packages/py-netcdf4/package.py

```diff
@@ -57,6 +55,13 @@ class PyNetcdf4(PythonPackage):
     # following patch disables the usage of pkg-config at all.
     patch("disable_pkgconf.patch")
 
+    # Allow building py-netcdf4 ~mpi when netCDF was build with +mpi. This patch
+    # overrides the auto-decect feature (has_parallel_support) in setup.py. The
+    # logic in setup.py changed between 1.6.5 and 1.7.1, therefore this patch
+    # only works for versions 1.7.1 and later.
+    # https://github.com/Unidata/netcdf4-python/issues/1389
+    patch("nompi.patch", when="@1.7.1: ~mpi")
```

Comment:
- Can you, please, rename the patch to something like `disable_parallel_support.patch` for consistency with the other one?
- The patch should be applicable when `@1.7:`.
- Probably, you can extend the `when` argument with `^netcdf-c+mpi`.